packages/types/src/util.ts

```diff
@@ -54,7 +54,7 @@ export interface BodyLengthCalculator {
 }
 
 // TODO Unify with the types created for the error parsers
```

Comment:
This line can be removed

Reply:
was removed in https://github.com/aws/aws-sdk-js-v3/pull/1238